### ISSUE TITLE
Darktable unable to detect known camera - Mac

### ISSUE THREAD

**Issue by johans3** [darktable-org/darktable#2228]
Darktable is unable to detect my Canon Eos 600 and any hub able to read my camera memory card too. 
The refresh button to check for new devices should help but doesn't, unfortunately. 

Both my hub and my camera are instantly recognized by my Mac OsX system
I use the latest public version of Darktable available for my system

**Comment by johans3** [comment]
I don't write bugs to degrade darktable. 
If your software isn't able to load import pictures from a well known camera i'm forced to do this trough other software. 

The point is your software offers this feature but it doesn't work. I described above. 
I should read the manual to make it work? 
That's ridiculous. It doesn't work on Mac that's the truth. 
Therefore what's the reason to close this report? Antipathy, because I send many reports in a short time?

**Comment by parafin** [comment]
Bug tracker is not a user support forum. Use mailing list or discuss.pixls.us forum for that. But first read documentation, specifically https://www.darktable.org/about/faq/ covers this problem. Yes, you are supposed to read documentation if you encounter a problem. The fact that you don't want to do it is not a darktable's problem/bug. You're just wasting (limited) time of developers and maintainers.

**Comment by johans3** [comment]
The work around of your FAQ is outdated and doesn't work. And this is true already for a very long time because I tried this already last year. At that time I discovered darktable but hadn't yet use for it

Killing the demon responsible for the picture transfer doesn't help either. 
The refresh button gives no results not with or without running demon, but my platform sees the plugged devices (hub/camera). 

It's a platform issue and you're on Linux. What works for you doesn't for me, which is to consider.
